### PR TITLE
Add and use prettier config

### PR DESCRIPTION
**Pull request by lwassermann**
`prettier/prettier` should already be activated by extending prettier/recommended:
👉 https://github.com/prettier/eslint-plugin-prettier/blob/master/eslint-plugin-prettier.js#L113

The prettier config was mentioned in the comment, but not actually used.

(Also format the json using prettier 😄 )

### CODE REVIEW

File: templates/.eslintrc.json [hunk 1]
```diff
@@ -22,13 +22,10 @@
   "extends": [
     "airbnb",
     "plugin:jest/recommended",
-    "plugin:prettier/recommended"
+    "plugin:prettier/recommended",
+    "prettier",
```
Comment (pmeinhardt) [inline templates/.eslintrc.json]: Seems like we don't really need this one?

https://github.com/prettier/eslint-plugin-prettier/blob/master/eslint-plugin-prettier.js#L110-L111

File: templates/.eslintrc.json [hunk 2]
```diff
@@ -22,13 +22,10 @@
...
+    "prettier/react"
   ],
-  "plugins": [
-    "jest",
-    "prettier"
-  ],
-  "rules": {
-    "prettier/prettier": "error"
-  }
+  "plugins": ["jest", "prettier"],
```
Comment (pmeinhardt) [inline templates/.eslintrc.json]: Same here, looks like `plugin:prettier/recommended` already takes care of this?

Comment (pmeinhardt) [reply templates/.eslintrc.json]: And I guess we could even drop `"jest"` from the plugins list?

https://github.com/jest-community/eslint-plugin-jest/blob/master/src/index.ts#L42

**Comment by lwassermann** [comment]
🤔 Now that we've taken almost everything out, I have the feeling it is much less clear what is happening.

**Comment by pmeinhardt** [comment]
Yep. I also feel like the more explicit version is somewhat clearer albeit being a little more verbose/redundant.

**Comment by lwassermann** [comment]
Interestingly, I have the sense that I do not mind duplication for the extends and plugins, but for the rules I disliked it. 🤷‍♂

**Comment by lwassermann** [comment]
I've added back the plugins and duplicate extends, for consistency. Please re-approve if you agree.

**Comment by pmeinhardt** [comment]
Oops. Totally overlooked this one here. 🙈 I'm sorry @lwassermann.
I think we have everything here covered on [`master`](https://github.com/bitcrowd/javascript/blob/master/templates/.eslintrc.json) now, don't we?

**Pull request closed without merging**